packages/round-manager/src/test-utils.tsx

```diff
@@ -68,3 +66,39 @@ export const makeStubRound = (overrides: Partial<Round> = {}): Round => {
     ...overrides
   }
 }
+
+export const makeStubApplication = (overrides: Partial<GrantApplication> = {}): GrantApplication => {
+  return {
+    id: faker.datatype.uuid(),
+    round: faker.finance.ethereumAddress(),
+    recipient: faker.finance.ethereumAddress(),
+    project: {
+      id: faker.datatype.uuid(),
+      website: faker.internet.url(),
+      logoImg: faker.image.imageUrl(),
+      metaPtr: {
+        protocol: faker.datatype.number(),
+        pointer: faker.finance.ethereumAddress()
+      },
+      title: faker.name.middleName(),
+      description: faker.name.jobDescriptor(),
+      lastUpdated: faker.date.recent(5).getTime(),
+    },
+    answers: [],
+    projectsMetaPtr: {
+      protocol: faker.datatype.number(),
+      pointer: faker.datatype.uuid()
+    },
+    ...overrides
+  }
+}
+
+export const renderWrapped = (ui: JSX.Element) => {
+  render(
```

Comment:
`render` is already imported as `rtlRender`, we should pick one or the other
```suggestion
  rtlRender(
```